sda-commons-dependency-check/src/test/java/org/sdase/commons/dependency/check/AvoidJavaxTest.java

```diff
@@ -0,0 +1,58 @@
+package org.sdase.commons.dependency.check;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.github.classgraph.ClassGraph;
+import io.github.classgraph.ScanResult;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class AvoidJavaxTest {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AvoidJavaxTest.class);
+
+  final Set<String> ALLOWED_FILE_PARTS =
+      Stream.of(
+              // Wiremock repacked classes. A Jakarta variant is not available yet. This will not
+              // influence production code.
+              "/com.github.tomakehurst/wiremock-jre8-standalone/",
```

Comment:
Is this still the correct path? I noticed that you switched to `org.wiremock:wiremock:3.3.1`.

Reply:
If we don't use the standalone variant any more, this exclude should not be needed.

Reply:
It was removed, indeed